### ISSUE TITLE
Accessibility adjustments

### ISSUE THREAD

**Issue by zepich** [mosparo/mosparo#194]
### Explanation

We received some feedback regarding the accessibility of the frontend box and will optimize mosparo accordingly.

### Changes

- [x] Add a hint to the main label that the form is protected by mosparo, visible only for screen readers. This is only added if the logo is visible on hover. If the project does not display the mosparo logo, the additional text will not be added.
- [x] Adjust the processing sentence to "Our spam protection verifies your data. Please wait...". The sentence right now is not easy to understand with a screen reader, especially in the Invisible mode where the screen reader doesn't read the main label.
- [x] Link alert message with the label. Right now, the alert message is not connected to the label, which can cause troubles for screen reader users.

### Discarded changes

- Test the alert messages with iOS VoiceOver. Since the alerts got replaced too fast, iOS skips reading them. 
   **NOT POSSIBLE, see here: https://github.com/w3c/aria/issues/1689**
